e2e/validators.go

```diff
@@ -1201,3 +1201,64 @@ func ValidateMIGInstancesCreated(ctx context.Context, s *Scenario, migProfile st
 	require.NotContains(s.T, stdout, "No MIG-enabled devices found", "no MIG devices were created.\nOutput:\n%s", stdout)
 	s.T.Logf("MIG instances with profile %s are created", migProfile)
 }
+
+func execOnPodForScenario(ctx context.Context, s *Scenario, pod *corev1.Pod, cmd string) *podExecResult {
```

Comment:
we already have a number of funcs that do this, I don't really see the value in adding another super thin helper like this